databuilder/extractor/dashboard/mode_analytics/mode_dashboard_owner_extractor.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+from typing import Any  # noqa: F401
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.dashboard.mode_analytics.mode_dashboard_utils import ModeDashboardUtils
+from databuilder.extractor.restapi.rest_api_extractor import MODEL_CLASS
+from databuilder.rest_api.rest_api_failure_handlers import HttpFailureSkipOnStatus
+from databuilder.rest_api.rest_api_query import RestApiQuery
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ModeDashboardOwnerExtractor(Extractor):
+    """
+    A Extractor that extracts Dashboard owner.
+
+    """
+
+    def init(self, conf):
+        # type: (ConfigTree) -> None
+        self._conf = conf
+
+        restapi_query = self._build_restapi_query()
+        self._extractor = ModeDashboardUtils.create_mode_rest_api_extractor(
+            restapi_query=restapi_query,
+            conf=self._conf.with_fallback(
+                ConfigFactory.from_dict(
+                    {MODEL_CLASS: 'databuilder.models.dashboard.dashboard_owner.DashboardOwner', }
```

Comment:
will it create new owner if the owner doesn't exists? or just create the relationship? or it will update the existing owner even the owner user does exists.

Reply:
It will `just create the relationship`. 
Thought about it a bit and creating owner implicitly means that it's not been populated by `Amundsen people` and probably better to skip it instead of creating it with just `email`